utils/apiFunctional.cjs

```diff
@@ -68,7 +69,8 @@ const trigger = function(gatewayUrl, username, password, team_name, project_name
         method: 'POST',
         headers: {
             'Content-Type': 'application/json'
-        }
+        },
+        rejectUnauthorized: false
```

Comment:
Why are we using this flag?

Reply:
was getting self signed certificate in certificate chain error when ever we try to trigger Unum instance , so we used it